### PR TITLE
Add Historic Datasets to Green Fast Track

### PR DESCRIPTION
**Pull request by alexrichey**
Adds a few Green Fast Track datasets, and fixes some issues with the ESRI connector. (specifics in the commit message bodies) 

Most importantly, feeding geojson with mixed case column names to gdal causes those columns to be null in the output format.

### CODE REVIEW

File: dcpy/connectors/esri/arcgis_feature_service.py [hunk 1]
```diff
@@ -28,12 +30,17 @@ class FeatureServer(BaseModel, extra="forbid", use_enum_values=True):
...
+        server_id = servers[self.server]["id"]
+        return f"https://{subdomain}.arcgis.com/{server_id}/ArcGIS/rest/services/{self.name}/FeatureServer/{self.layer}"
 
 
 def get_metadata(dataset: FeatureServer) -> dict:
     resp = requests.get(f"{dataset.url}", params={"f": "pjson"})
     resp.raise_for_status()
+    error = resp.json().get("error")  # 200 responses might contain error details
+    if error:
+        raise Exception(f"Error fetching ESRI Server metadata: {error}")
```
Comment (fvankrieken) [inline dcpy/connectors/esri/arcgis_feature_service.py:43]: Thank you

**Review by fvankrieken — APPROVED** [review]
Looks great. All library datasets archiving successfully?

**Comment by alexrichey** [comment]
> Looks great. All library datasets archiving successfully?

@fvankrieken Yeah, kind of. It's certainly improved. So here's an example: the [nysshpo_archaeological_buffer_areas](https://cloud.digitalocean.com/spaces/edm-recipes?path=datasets%2Fnysshpo_archaeological_buffer_areas%2F20240314%2F&i=266877). The one outstanding issue (if it's an issue at all) is that the csv's don't seem to have the geometry column. Other formats look good.

**Comment by fvankrieken** [comment]
That's needing 
```yml
destination:
    options:
      - "GEOMETRY=AS_WKT"
 ```

**Pull request merged**